lib/perl/Genome/InstrumentData/AlignmentResult/Command/BamToCram.pm

```diff
@@ -0,0 +1,47 @@
+package Genome::InstrumentData::AlignmentResult::Command::BamToCram;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::InstrumentData::AlignmentResult::Command::BamToCram {
+    is => 'Genome::InstrumentData::AlignmentResult::Command::ConvertBase',
+};
+
+sub shortcut {
+    my $self = shift;
+
+    return $self->_is_currently_cram;
+}
+
+sub execute {
+    my $self = shift;
+
+    my $guard = $self->_lock()->unlock_guard;
+
+    return 1 if $self->_is_currently_cram;
+
+    unless($self->_is_currently_bam) {
+        $self->fatal_message("This result cannot be converted to CRAM.");
```

Comment:
I'd suggest: "This result is already in CRAM format!", which seems a little more transparent to the end user.